### ISSUE TITLE
Suggestions to add synonyms

### ISSUE THREAD

**Issue by drseb** [obophenotype/human-phenotype-ontology#4936]
@psergouniotis 

should these synonyms go into HPO somewhere?

- central cloudy dystrophy
- central cloudy dystrophy of Francois
- crocodile shagreen
- fleck corneal dystrophy
- François-Neetens fleck corneal dystrophy

if so, where?

**Comment by pnrobinson** [comment]
I think these are all diseases except for crocodile shagreen
I will add a term for that. @psergouniotis please shout if you disagree.
https://www.ncbi.nlm.nih.gov/pubmed/19682623

**Comment by pnrobinson** [comment]
@psergouniotis 
-- is the following a good term?
Label: posterior crocodile shagreen of the cornea
Def: Grayish, polygonal pattern of opacities with intervening clear zones across the central cornea that resembles crocodile skin.
https://www.ncbi.nlm.nih.gov/pubmed/19682623

child of : https://hpo.jax.org/app/browse/term/HP:0007881

**Comment by pnrobinson** [comment]
adding Posterior crocodile shagreen of the cornea
